phoenix-core/src/main/java/org/apache/phoenix/schema/PTable.java

```diff
@@ -104,8 +104,9 @@ public ViewType combine(ViewType otherType) {
     }
 
     public enum IndexType {
-        GLOBAL((byte)1),
-        LOCAL((byte)2);
+        GLOBAL((byte)1), // Covered Global
+        LOCAL((byte)2), // Covered Local
+        UNCOVERED((byte)3); // Uncovered Global
```

Comment:
Since we can do uncovered queries on local indexes, would it be more clear if the enum were UNCOVERED_GLOBAL (or GLOBAL_UNCOVERED)?

Reply:
I will change the enum to UNCOVERED_GLOBAL as you suggested.